app/models/course/forum/topic.rb

```diff
@@ -99,4 +104,12 @@ def generate_initial_post
   def set_initial_post_title
     posts.first.title = title
   end
+
+  def mark_as_read_for_creator
+    mark_as_read! for: creator
+  end
+
+  def mark_as_read_for_updater
+    mark_as_read! for: updater
```

Comment:
We use without brackets in other places.  

Actually shall we use the suggestion from this style guide ? : https://github.com/airbnb/ruby#no-return-parens